### PR TITLE
Supported customizable url for csgo-league-web

### PR DESCRIPTION
**Pull request by FlowingSPDG**
and edited README.md for installation section

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "token": "",
   "colour": "0xffffff",
-  "prefix": "!"
+  "prefix": "!",
+  "web": "http://localhost:5000"
```
Comment (B3none) [inline config.json]: Could we switch this to `url` instead? Seems a bit less ambiguous in my opinion

File: src/commands/login.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ module.exports = {
   command: (client, message) => {
     let author = message.author;
 
-    axios.get(`https://league.redlinecs.net/discord/generate/${author.id}`)
+    axios.get(`${config.web}/discord/generate/${author.id}`)
```
Comment (B3none) [inline src/commands/login.js]: Switch to `config.url`

File: src/commands/logout.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ module.exports = {
   command: (client, message) => {
     let author = message.author;
 
-    axios.get(`https://league.redlinecs.net/discord/${author.id}`)
+    axios.get(`${config.web}/discord/${author.id}`)
```
Comment (B3none) [inline src/commands/logout.js]: Switch to `config.url`

File: src/commands/logout.js [hunk 4]
```diff
@@ -33,7 +33,7 @@ module.exports = {
               fields: [
                 {
                   name: 'Steam login URL:',
-                  value: `https://league.redlinecs.net/discord/${author.id}/${code}`
+                  value: `${config.web}/discord/${author.id}/${code}`
```
Comment (B3none) [inline src/commands/logout.js]: Switch to `config.url`

**Review by B3none — CHANGES_REQUESTED** [review]
Looks good but I'd like to see a small change please! 👍

**Comment by FlowingSPDG** [comment]
replaced `config.web` to `config.url`,done :D

**Pull request merged**